test/scripts/console/deploy.js

```diff
@@ -18,6 +18,12 @@ describe('deploy', () => {
 
   after(() => fs.rmdir(hexo.base_dir));
 
+  it('no deploy config', () => {
+    delete hexo.config.deploy;
+
+    return deploy({test: true});
```

Comment:
What is the expected return value of `deploy()`, after it resolved?
suggest to refactor to async/await (including other tests in this PR).

Reply:
> What is the expected return value of `deploy()`, after it resolved?

No value at all. When there is no config, `console.log()` will be used to display an error message.